xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1303,7 +1303,7 @@ void CRenderManager::PrepareNextRender()
       m_QueueSkip++;
     }
 
-    int lateframes = (renderPts - m_Queue[idx].pts) / frametime;
+    int lateframes = (renderPts - m_Queue[idx].pts) * (m_fps * DVD_TIME_BASE);
```

Comment:
In fact as we discussed on IRC:

A frame is: (1 / m_fps) * DVD_TIME_BASE in length (pts)
So here we need: (r - i) / ((1 / m_fps) * DVD_TIME_BASE) == (r -i) / (DVD_TIME_BASE / m_fps) == (r - i) * m_fps / DVD_TIME_BASE